src/neptune/api/models.py

```diff
@@ -150,7 +152,17 @@ def from_model(model: Any) -> Field:
     @staticmethod
     def from_proto(data: Any) -> Field:
         field_type = str(data.type)
-        return Field._registry[field_type].from_proto(data.__getattribute__(f"{camel_to_snake(field_type)}_properties"))
+
+        try:
+            # We might receive a field that is not (fully) supported
+            if field_type not in Field._registry:
```

Comment:
Instead of `try..catch` you should extend this if statement with `hasattr` and just return `UnknownField`